### PR TITLE
Uses value instead of placeholder for default val

### PR DESCRIPTION
**Pull request by nesalang**

### CODE REVIEW

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -389,6 +397,16 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		F4071C751FCCBAEF00AF4FEA /* json */ = {
+			isa = PBXGroup;
+			children = (
+				F4071C761FCCBAEF00AF4FEA /* json-forwards.h */,
```
Comment (riarenas) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj]: Why is this needed now?

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDateTextField.mm [hunk 2]
```diff
@@ -25,10 +25,12 @@ - (instancetype)initWithTimeDateInput:(std::shared_ptr<BaseInputElement> const &
 
     if(self)
     {
+        NSString *valueStr = nil;
         NSString *placeHolderStr = nil;
         NSString *minDateStr = nil;
         NSString *maxDateStr = nil;
         
+
```
Comment (riarenas) [inline source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRDateTextField.mm]: [nit] extra line

File: source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nesalang) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj]: it was always there; i just changed directory representation in xcode solution file

---
In reply to: [153350249](https://github.com/Microsoft/AdaptiveCards/pull/992#discussion_r153350249) [](ancestors = 153350249)

Comment (riarenas) [reply source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj]: Ahh ok then 😄
